Gir_GObject.toml

```diff
@@ -0,0 +1,25 @@
+[options]
+girs_dir = "gir-files"
+library = "GObject"
+version = "2.0"
+min_cfg_version = "2.32"
+target_path = "."
+auto_path = "src/gobject/auto"
+work_mode = "normal"
+single_version_file = true
+
+generate = [
+    "GObject.Binding",
+    "GObject.BindingFlags",
+]
+
+ignore = [
+]
+
+manual = [
+    "GObject.Object",
+]
+
+[[object]]
+name = "GObject.*"
+status = "ignore"
```

Comment:
In final version this need be status = "generate"

Reply:
Why? I don't care about any of those functions. They can be added when someone needs them :)

And more important, all that can be autogenerated need manual bindings (and many of them have already, either here or in my `gobject-subclass` crate)

Reply:
Ok